tests/replicaset/manager-getservers-001.phpt

```diff
@@ -158,7 +158,7 @@ array(3) {
       ["primary"]=>
       string(19) "192.168.112.10:3000"
       ["tags"]=>
-      object(stdClass)#%d (%d) {
+      array(2) {
```

Comment:
This also looks suspicious. Why are things changing from stdclass->array?
The ticket is about the top-level array become a stdclass.....